### ISSUE TITLE
a11y improvements for header

### ISSUE THREAD

**Issue by maria-jon** [openaq/openaq.org#1120]
Header can improve in accessibility in a few places: 

### Hamburger button in nav 

- Missing :focus state   
- Missing aria-label  
- Missing alt text  
    
### Keyboard navigation
- Unable to tab to hamburger
